### PR TITLE
add Manchvegas recommendations

### PR DESCRIPTION
**Pull request by jackief**
maybe @chuckp01 can accept mine or @TerriLLBean 

this pull request closes issue #19

### CODE REVIEW

File: jackief-manchvegas/things-to-do.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Restaurants
+
+Things to do
```
Comment (pgrech98) [inline jackief-manchvegas/things-to-do.md]: Good work...

Comment (smithmmf) [inline jackief-manchvegas/things-to-do.md]: What about the tacos?

Comment (jackief) [inline jackief-manchvegas/things-to-do.md]: Good idea @smithmmf I will add that

**Comment by smithmmf** [comment]
I needed to comment!  OOPS!

**Pull request merged**